packages/expo-modules-core/src/NativeModulesProxy.native.ts

```diff
@@ -2,10 +2,10 @@ import { NativeModules } from 'react-native';
 
 import { ProxyNativeModule } from './NativeModulesProxy.types';
 
+const LegacyNativeProxy = NativeModules.NativeUnimoduleProxy;
 // Fixes `cannot find name 'global'.` in tests
 // @ts-ignore
 const ExpoNativeProxy = global.ExpoModules?.NativeModulesProxy;
-const LegacyNativeProxy = NativeModules.NativeUnimoduleProxy;
```

Comment:
Why such re-order? 🤔

Reply:
Ohh, sorry, I forgot to leave a comment here. This re-order is needed because by accessing the legacy native proxy, we're installing the jsi interop. So before the `global.ExpoModules` will be undefined.